README.md

```
@@ -45,11 +45,11 @@
 ## Active Clients
 
 <a name="kawata-aeris"></a>
-### [Kawata](https://kawata.pw) / Aeris
+### Aeris - [Kawata](https://kawata.pw) <a href="https://kawata.pw"><img src="public/kawata.png" alt="Kawata mark" width="25" align="middle"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)